### PR TITLE
stp: new home on GitHub

### PR DESCRIPTION
**Pull request by L2G**
I found STP's new home by following the link from the original home.

### CODE REVIEW

File: stp.rb [hunk 1]
```diff
@@ -1,11 +1,9 @@
-require 'formula'
-
 class Stp < Formula
-  head 'http://stp-fast-prover.svn.sourceforge.net/svnroot/stp-fast-prover/trunk/stp', :revision => '1134'
-  homepage 'http://sites.google.com/site/stpfastprover/'
+  head "https://github.com/stp/stp.git"
+  homepage "https://stp.github.io/"
```
Comment (DomT4) [inline stp.rb]: Can you flip these around so `head` is below `homepage`? Thanks!

If you want to add a `desc` as well, that'd be cool.

Comment (L2G) [inline stp.rb]: Will do.

**Comment by L2G** [comment]
STP has apparently been rewritten to use CMake, so more changes to the formula are in the works… :scream_cat:

**Comment by bfontaine** [comment]
Ping?

**Comment by bfontaine** [comment]
Closing due to no response. Feel free to open another PR that addresses the CMake issue as well as Dom’s comment. Thanks!

**Pull request closed without merging**